### PR TITLE
Add iframe to embed technical roadmap

### PR DESCRIPTION
**Pull request by garethbowen**
medic/cht-docs#490

### CODE REVIEW

**Comment by garethbowen** [comment]
@abbyad Review please! I'm not sure this will actually work - I couldn't find any prior art, and don't have a local install running to test it. Does it look right to you?

File: content/en/core/overview/technical-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Technical roadmap"
+linkTitle: "Technical roadmap"
```
Comment (abbyad) [inline content/en/core/overview/technical-roadmap.md:3]: Updated to title case, as per https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#grammar-and-punctuation-in-headers

```suggestion
title: "Technical Roadmap"
linkTitle: "Technical Roadmap"
```

File: content/en/core/overview/technical-roadmap.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+weight: 2
+description: >
+  Outlines the planned changes to technical aspects of the CHT.
```
Comment (abbyad) [inline content/en/core/overview/technical-roadmap.md:6]: Would be good to keep punctuation and phrase style consistent with other pages. This isn't explicitly stated in the style guide, so I'll add it.

```suggestion
  Planned changes to technical aspects of the CHT
```

File: content/en/core/overview/technical-roadmap.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+---
+
+<iframe src="https://technicalroadmap.communityhealthtoolkit.org" frameborder="0"></iframe>
```
Comment (abbyad) [inline content/en/core/overview/technical-roadmap.md:9]: Currently shows up like this:

image

The width can be set to 100% and works well, but the height will be a little trickier.

**Review by abbyad — CHANGES_REQUESTED** [review]
Some changes requested inline, but I am curious about the objective of having the roadmap for the intended audience. For instance, will items in the roadmap help them know about changes required to apps, and do Now or Done items need to be associated with particular versions to be helpful? Also, since the iframe content loads after the main page content, stating the objective more clearly in the page would also help to bring context to the page while it loads.

The [local setup for the doc site](https://github.com/medic/cht-docs#installing-hugo) is pretty quick, and it would be good to know if you have difficulties with it.

File: content/en/core/overview/technical-roadmap.md (hunk 3 above)
Comment (garethbowen) [reply content/en/core/overview/technical-roadmap.md]: 100% width and 500px height picked.

**Review by abbyad — APPROVED** [review]
Looks great!

As it grows it may be helpful to make it easier to see the roadmap in a separate tab/page, but that can come later.

**Pull request merged**